app/index.html

```diff
@@ -7,6 +7,11 @@
 </head>
 <body class="container">
 <center><h1>Oasis Secret Ballot dApp</h1></center>
+<center>
+    <div id="warning-div">
+        <p>Reminder: Please ensure you are using the Oasis Devnet (configured on MetaMask) and received DEV from the <a href="https://faucet.oasiscloud.io/">Oasis Faucet</a>.</p>
```

Comment:
slight bikeshed: maybe we should link to the tutorial instead of directly to the faucet

Reply:
Yeah, I was slightly worried about this; happy to fix that.